### PR TITLE
Add screaming sounds for a larva bursting out of a marine + bursting animations

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
## About the PR
Larvas bursting out of a marine now has a 3 second do-after
During the do-after the host will be screaming in pain

## Why / Balance
Cool, CM13 parity

## Media

https://github.com/user-attachments/assets/17a258ce-c9f4-4302-8048-023d642be598

https://github.com/user-attachments/assets/d6836965-1162-433d-8bb2-f1215e01a277

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Content.Shared/_RMC14/Xenonids/Parasite/InfectableComponent.cs [hunk 1]
```diff
@@ -13,4 +13,11 @@ public sealed partial class InfectableComponent : Component
         [Sex.Male] = new SoundPathSpecifier("/Audio/_RMC14/Voice/Human/infected_male.ogg"),
         [Sex.Female] = new SoundPathSpecifier("/Audio/_RMC14/Voice/Human/infected_female.ogg")
     };
+
+    [DataField, AutoNetworkedField]
+    public Dictionary<Sex, SoundSpecifier> PreburstSound = new()
+    {
+        [Sex.Male] = new SoundCollectionSpecifier("RMCMalePreburstScreams", AudioParams.Default.WithVariation(0.05f)),
+        [Sex.Female] = new SoundCollectionSpecifier("RMCFemalePreburstScreams", AudioParams.Default.WithVariation(0.05f))
```
Comment (DrSmugleaf) [inline Content.Shared/_RMC14/Xenonids/Parasite/InfectableComponent.cs:21]: Is the variation here intended? Whisper thinks it makes the screams sound strange

File: Content.Shared/_RMC14/Xenonids/Parasite/SharedXenoParasiteSystem.cs [hunk 2]
```diff
@@ -476,8 +479,9 @@ public override void Update(float frameTime)
                         var message = Loc.GetString("rmc-xeno-infection-insanepain-" + random);
                         _popup.PopupEntity(message, uid, uid, PopupType.LargeCaution);
 
-                        var knockdownTime = infected.BaseKnockdownTime * 10;
-                        InfectionShakes(uid, infected, knockdownTime, knockdownTime, false);
+                        var knockdownTime = infected.BaseKnockdownTime * 2;
+                        var jitterTime = infected.JitterTime * 0;
```
Comment (DrSmugleaf) [inline Content.Shared/_RMC14/Xenonids/Parasite/SharedXenoParasiteSystem.cs:483]: Are these changes intended? @Vermidia wrote the original ones, but they are hard to check in 13

File: Resources/Audio/_RMC14/Voice/Human/attributions.yml [hunk 3]
```diff
@@ -82,3 +82,24 @@
...
+  - female_preburst7.ogg
+  - male_preburst1.ogg
+  - male_preburst2.ogg
+  - male_preburst3.ogg
+  - male_preburst4.ogg
+  - male_preburst5.ogg
+  - male_preburst6.ogg
+  - male_preburst7.ogg
+  - male_preburst8.ogg
+  - male_preburst9.ogg
```
Comment (DrSmugleaf) [inline Resources/Audio/_RMC14/Voice/Human/attributions.yml:102]: I need to check if any of these are copyrighted but the ones in the video don't seem to be at least

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Content.Shared/_RMC14/Xenonids/Parasite/SharedXenoParasiteSystem.cs (hunk 2 above)
Comment (Dutch-VanDerLinde) [reply Content.Shared/_RMC14/Xenonids/Parasite/SharedXenoParasiteSystem.cs:483]: Yes, supposed to mimic the jitter starting up violently when the larva is bursting

File: Content.Shared/_RMC14/Xenonids/Parasite/InfectableComponent.cs (hunk 1 above)
Comment (Dutch-VanDerLinde) [inline Content.Shared/_RMC14/Xenonids/Parasite/InfectableComponent.cs:21]:
```suggestion
        [Sex.Male] = new SoundCollectionSpecifier("RMCMalePreburstScreams"),
        [Sex.Female] = new SoundCollectionSpecifier("RMCFemalePreburstScreams")
```

Comment (Dutch-VanDerLinde) [reply Content.Shared/_RMC14/Xenonids/Parasite/InfectableComponent.cs:21]: Removed

**Comment by Dutch-VanDerLinde** [comment]
image

added bursting animations aswell

**Review by DrSmugleaf — COMMENTED** [review]
Are these a new thing in 13? When I was playing the other day it didn't do it on spawn, idk if it doesn't do it for the on-spawn larva

**Comment by Dutch-VanDerLinde** [comment]
> Are these a new thing in 13? When I was playing the other day it didn't do it on spawn, idk if it doesn't do it for the on-spawn larva

it's a test merge i believe

**Comment by Dutch-VanDerLinde** [comment]
but the animation was broken on 13 and there was a pr to fix it over there

**Comment by Dutch-VanDerLinde** [comment]
#6824

**Comment by Dutch-VanDerLinde** [comment]
made the final jitter more violent

**Comment by Dutch-VanDerLinde** [comment]
Once this is merged i'll work on finishing larva surgery (fully grown larvas can be extracted, but wont kill the larva) as per the TODO list

**Comment by DrSmugleaf** [comment]
I'd rather wait for 13's PR to be merged fully before we get the sound specifically, feel free to disable that for now though and I can merge this

File: Resources/Audio/_RMC14/Voice/Human/attributions.yml [hunk 4]
```diff
@@ -82,3 +82,24 @@
...
+  license: "CC-BY-SA-3.0"
```
Comment (DrSmugleaf) [inline Resources/Audio/_RMC14/Voice/Human/attributions.yml:103]: I checked 13's PR and they are from TGMC which would make these NC
It's not clear if you can use NC assets in a game with a Patreon even if its a free to play game, so we wouldn't be able to use these

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

**Comment by Dutch-VanDerLinde** [comment]
> I'd rather wait for 13's PR to be merged fully before we get the sound specifically, feel free to disable that for now though and I can merge this

do I remove the sound files or just disable the system

**Comment by Dutch-VanDerLinde** [comment]
removed the sounds, commented out the related code

**Comment by Dutch-VanDerLinde** [comment]
added a new frame on the bursting animation, 2.4 delay so it's timed with the do-after

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**